### PR TITLE
[CI] Reuse fixture infrastructure for select-member tests

### PR DESCRIPTION
**Pull request by jonasjucker**

### CODE REVIEW

File: tests/helpers.py [hunk 1]
```diff
@@ -72,17 +75,20 @@ def run_performance_cli(timing_regex, timing_database):
...
         stats_file_name,
         "--tolerance-file-name",
         tolerance_file_name,
         "--member-num",
-        "10",
-        "--member-type",
-        "dp",
+        str(member_num),
     ]
+    if member_type:
```
Comment (huppd) [inline tests/helpers.py:89]: Maybe pedantic. But if I understood correctly, I would prefer:
```suggestion
    if member_type is not None:
```

File: tests/helpers.py [hunk 2]
```diff
@@ -175,11 +181,86 @@ def run_cdo_table_cli(model_output_dir, cdo_table_file, perturbed_model_output_d
...
+
+def catch_error(result):
     if result.exit_code != 0:
         error_message = "Error executing command:\n" + result.output
         if result.exception:
             error_message += "\nException: " + str(result.exception)
         raise Exception(error_message)
+
+
+def create_artificial_stats_file(filename, configurations, seed, perturbation):
```
Comment (huppd) [inline tests/helpers.py:233]: Just a matter of taste, I would prefer
```suggestion1
def create_random_stats_file(filename, configurations, seed, perturbation):
```
or 
```suggestion2
def create_stats_file_with_random_values(filename, configurations, seed, perturbation):
```
But open for other suggestion or convincing ;)

Comment (jonasjucker) [reply tests/helpers.py]: done

**Review by huppd — APPROVED** [review]
Thanks for doing the additional changes.

**Pull request merged**